purchases/src/main/kotlin/com/revenuecat/purchases/Offerings.kt

```diff
@@ -10,6 +10,11 @@ data class Offerings(
     val current: Offering?,
     val all: Map<String, Offering>,
 ) {
+    internal var placements: Placements? = null
+
+    internal constructor(current: Offering?, all: Map<String, Offering>, placements: Placements?) : this(current, all) {
```

Comment:
Oh just saw you doing this. I would make this the other way instead. Something like this:
```
data class Offerings internal constructor(
    val current: Offering?,
    val all: Map<String, Offering>,
    internal val placements: Placements? = null,
) {
   constructor(current: Offering?, all: Map<String, Offering>): this(current, all, null))
   ....
}
```
That way it's clearer to see all the parameters in the main constructor which is internal.